dashboard/src/components/util/LastEnvironmentRedirect.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { compose } from "recompose";
+import { Redirect, withRouter } from "react-router-dom";
+import { graphql, Query } from "react-apollo";
+import gql from "graphql-tag";
+
+const primaryQuery = gql`
+  query LastEnvironmentQuery {
+    lastEnvironment @client {
+      organization
+      environment
+    }
+  }
+`;
+
+const fallbackQuery = gql`
```

Comment:
Would it make sense to combine these into a single query? I don't suspect there is any real perf advantage of splitting them.

Reply:
One hits the network and the other doesn't.